src/component/app/index.tsx

```diff
@@ -232,14 +220,25 @@ function VentureRoutes(props: VentureRoutesProps) {
 
   const venturesLoaded = ventureUserSuccess && ventureSuccess;
 
-  const currentVenture = ventureSlug
-    ? allVentures?.filter(
-        (venture: IVenture) =>
-          venture.name.toLowerCase().replace(/\s/g, "") === ventureSlug
-      )[0]
-    : ventureSuccess && ventureUserSuccess
-    ? allVentures[0]
-    : undefined;
+  function calcVentureSlug(venture: IVenture): string {
+    return venture.name.toLowerCase().replace(/\s/g, "");
+  }
+
+  let currentVenture: IVenture | undefined;
+  if (ventureSlug) {
+    currentVenture = allVentures?.find(
+      (venture: IVenture) => calcVentureSlug(venture) === ventureSlug
+    );
+  } else if (ventureSuccess && ventureUserSuccess) {
+    const sortedVentures = [...allVentures].sort((a: IVenture, b: IVenture) => {
```

Comment:
This ensures that when a user is redirected from `/` to `/<first venture>`, they always go to the same venture after reloading. Currently it is random which one they will go to because they are not sorted.